Alejandroq12/my-books-ES6
# Restructure My Books App using ES6 and Modular Approach

Dear reviewer.
Thank you for your valuable time.

# Description:
- In this pull request, I have restructured the Awesome Books app to utilize ES6 syntax and a modular approach. The project now follows best practices for HTML, CSS, and JavaScript, and uses Luxon for displaying the current date.

# Key Features:

- ES6 syntax for cleaner and more efficient code
- Modular approach for better organization and maintainability
- Utilization of Luxon library for date and time display
- Compliance with HTML, CSS, and JavaScript best practices

# Changes Made:

- Created a new directory and initialized it with git
- Set up package.json file using npm init -y
- Created index.js as the entry point for JavaScript code
- Created index.css for styles
- Created index.html and linked index.js and index.css
- Created a modules directory for modular organization
- Refactored code to use ES6 syntax and modular structure
- Converted functions to arrow functions
- Utilized let and const appropriately
- Displayed current date below the navigation bar using Luxon library

# Testing:

- Ensure there are no linter errors
- Verify correct usage of GitHub Flow
- Check for professional documentation of work
- Estimated time spent: 4.5 hours

Please note that while working on this project, I had to use Webpack in a **separate branch** only for deploying the site. This was necessary to ensure that Luxon works correctly when the site is deployed. **This approach allowed me to keep this branch focused only on the specific requirements of the project as it very important to meet the exact requirements.**

You can view the deployed site here: https://alejandroq12.github.io/my-books-ES6/

Thank you for your understanding and review.
Have a very good day.
![HappyBirthdayGIF](https://user-images.githubusercontent.com/95459740/228087553-8276a094-5b9f-4b0d-9de4-5a2cf054c6f2.gif)
